src/components/MessagesList/MessagesGroup/Message/MessagePart/FilePreview.spec.js

```diff
@@ -51,14 +51,16 @@ describe('FilePreview.vue', () => {
 
 		propsData = {
 			token: 'TOKEN',
-			id: '123',
-			name: 'test.jpg',
-			path: 'path/to/test.jpg',
-			size: '128',
-			etag: '1872ade88f3013edeb33decd74a4f947',
-			permissions: '15',
-			mimetype: 'image/jpeg',
-			previewAvailable: 'yes',
+			file: {
+				id: '123',
+				name: 'test.jpg',
+				path: 'path/to/test.jpg',
+				size: '128',
+				etag: '1872ade88f3013edeb33decd74a4f947',
+				permissions: '15',
+				mimetype: 'image/jpeg',
+				previewAvailable: 'yes',
```

Comment:
```suggestion
				'preview-available': 'yes',
```
and one more in `test('renders generic mime type icon for unknown mime types'`